tests/views/test_message.py

```diff
@@ -509,13 +568,16 @@ def test_conversation_key_error(self, mock_request):
         self.assertEqual(len(request_history), 1)
         self.assertEqual(response.status_code, 500)
 
-    def test_get_radio_buttons(self):
+    @requests_mock.mock()
+    def test_get_radio_buttons(self, mock_request):
+        mock_request.get(url_get_surveys_list, json=survey_list)
         response = self.client.get("/messages/select-survey")
 
         self.assertEqual(200, response.status_code)
         self.assertIn("Filter messages by survey".encode(), response.data)
         self.assertIn("ASHE".encode(), response.data)
-        self.assertIn("Bricks".encode(), response.data)
```

Comment:
any reason for taking Bricks out?

Reply:
I must have just accidentally took it out without realising and added BRES to it. Can add it back if need be but i don't think its makes a difference